### PR TITLE
Adding release-drafter

### LINKED ISSUE #463: Add Release Drafter to neurolab
**Describe the issue**
Add Release Drafter to Neurolab

### PR DESCRIPTION
**Pull request by Abhik1998**
Fixes #463 

**Changes**: Added the GitHub app to this repository following this link https://github.com/apps/release-drafter

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [ ] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [ ] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

### CODE REVIEW

**Comment by adityastic** [comment]
@Abhik1998 you need to understand what I'm saying there.

**Comment by adityastic** [comment]
@jddeep add release drafter to the repo, then merge this

**Review by adityastic — CHANGES_REQUESTED** [review]
Squash into a single commit

**Comment by adityastic** [comment]
Also, use this file as a template: https://github.com/fossasia/badge-magic-android/blob/development/.github/release-drafter.yml

**Comment by Abhik1998** [comment]
@adityastic I used that .

**Comment by adityastic** [comment]
@Abhik1998 please check the file I linked to. It's different

**Review by adityastic — CHANGES_REQUESTED** [review]
Squash into a single commit

File: .github/release-drafter.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+tag-template: 'v$NEXT_PATCH_VERSION'
+categories:
+  - title: '🚀 Features'
+    labels:
+      - 'feature'
+      - 'enhancement'
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'fix'
+      - 'bugfix'
```
Comment (CloudyPadmal) [inline .github/release-drafter.yml]: Only use labels that are available (https://github.com/fossasia/neurolab-android/labels)

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]

**Comment by jddeep** [comment]
@Abhik1998 builds and checks are failing. Also, you didn't make the changes asked. Any updates on this?

**Comment by Abhik1998** [comment]
> @Abhik1998 builds and checks are failing. Also, you didn't make the changes asked. Any updates on this?

Ohh sorry, I was at home so I was unable to work there. I have resolved the earlier issue. I used a label that was not recognized hence this was the error.

**Comment by Abhik1998** [comment]
@jddeep  please have a review

**Review by CloudyPadmal — APPROVED** [review]

**Review by adityastic — APPROVED** [review]

**Comment by adityastic** [comment]
@jddeep ready to merge :+1:

**Pull request merged**